packages/ds-component/src/types.ts

```diff
@@ -322,34 +322,106 @@ export type DSRow = DSRowValue[];
  */
 export type DSRowValue = string | number | boolean;
 
-export interface ConfigDataElement {
+export interface ConfigDataElementMetric {
   /**
    * The data element type to render.
    */
-  type: ConfigDataElementType;
+  type: ConfigDataElementType.METRIC;
   /**
-   * The ID of the data element.
+   * The ID of the metric.
    *
    * This should be a non-empty string with no spaces.
    */
   id: ConfigDataElementId;
   /**
-   * The tooltip or label for the data element.
+   * The tooltip or label for the metric.
    */
   label: string;
   /**
-   * The data options for the element.
+   * The data options for a Metric.
+   */
+  options: {
+    /**
+     * The minimum number of metrics supported.
+     */
+    min: number;
```

Comment:
technicaly min/max don't have to exist in the elements

Reply:
fixed.